lib/lrama/command.rb

```diff
@@ -5,65 +5,104 @@ class Command
     LRAMA_LIB = File.realpath(File.join(File.dirname(__FILE__)))
     STDLIB_FILE_PATH = File.join(LRAMA_LIB, 'grammar', 'stdlib.y')
 
-    def run(argv)
-      parser_options(argv)
+    def initialize(argv)
+      @logger = Lrama::Logger.new
+      @options = OptionParser.parse(argv)
+      @tracer = Tracer.new(@logger, **@options.trace_opts)
```

Comment:
Could you keep to pass `io` without `Logger` wrapper? I think it's difficult to determine what severity is best for trace because trace is enabled by separated option.

Reply:
Fix in 36a2638d78e68f2e2cdbb9db9f43857bd9317e78